python/cugraph/cugraph/sampling/uniform_neighbor_sample.py

```diff
@@ -148,6 +284,18 @@ def uniform_neighbor_sample(
                     Contains the offsets of each batch in the sampling result
     """
 
+    if batch_id_list is not None:
+        return uniform_neighbor_sample_legacy(
```

Comment:
Same ask here to raise a depractated warning.

Reply:
it should raise a warning.  That function is wrapped with the deprecated warning wrapper

Reply:
Ahh missed that.